.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+
+Tell us what this change does. If you're fixing a bug, please mention
+the github issue number.
```

Comment:
Is it worth saying that links to `metasploit-payloads` PRs are required if Meterpreter changes are related?